fe/fe-core/src/main/java/com/starrocks/sql/optimizer/Optimizer.java

```diff
@@ -289,6 +290,7 @@ void logicalRuleRewrite(Memo memo, TaskContext rootTaskContext) {
         ruleRewriteIterative(memo, rootTaskContext, new MergeProjectWithChildRule());
         ruleRewriteOnlyOnce(memo, rootTaskContext, new GroupByCountDistinctRewriteRule());
         ruleRewriteOnlyOnce(memo, rootTaskContext, new ReorderIntersectRule());
+        ruleRewriteIterative(memo, rootTaskContext, new RemoveAggregationFromAggTable());
```

Comment:
Maybe we can add `RuleSetType` Of MVRewrite?

Reply:
it seems not necessary to add a RuleSetType